src/Octoshift/GithubClient.cs

```diff
@@ -29,6 +30,8 @@ public GithubClient(OctoLogger log, HttpClient httpClient, string personalAccess
             }
         }
 
+        public virtual async Task<string> GetHeadersAsync(string url) => (await SendAsync(HttpMethod.Get, url, shouldEnsureSuccessStatusCode: false)).Content;
```

Comment:
This method is very confusing. It suggests that it is just going to get the headers of a response but it's not! it's just a `GET` that ignores the success status check. I can see that in the API the use case is just to get (more likely to check) the archive's url, based on [this](https://docs.github.com/en/rest/reference/migrations#download-an-organization-migration-archive).
So just to check if the url exists or not you can just make a request with the existing `GetAsync` and if case no errors just return `true`.

Reply:
Updated the naming